### PR TITLE
Redo add_expense command

### PR DESCRIPTION
**Pull request by lightz96**

### CODE REVIEW

File: src/main/java/thrift/logic/commands/RedoCommand.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ * Redo undone command in THRIFT.
+ */
+public class RedoCommand extends Command {
+
+    public static final String COMMAND_WORD = "redo";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Redo undone command"
+            + "Example: " + COMMAND_WORD;
+
+    public static final String MESSAGE_SUCCESS = "Redo successfully";
```
Comment (limerencee) [inline src/main/java/thrift/logic/commands/RedoCommand.java]: Maybe should change the wording to `Redo successful`, same for `Undo` -> `Undo successful`.

File: src/test/java/thrift/logic/commands/AddExpenseCommandTest.java [hunk 2]
```diff
@@ -162,6 +191,12 @@ public void setTransaction(Transaction target, Transaction updatedTransaction) {
             throw new AssertionError("This method should not be called.");
         }
 
+        @Override
+        public Transaction getLastTransactionFromThrift() {
+            throw new AssertionError("This method should not be called.");
+
```
Comment (limerencee) [inline src/test/java/thrift/logic/commands/AddExpenseCommandTest.java]: Extra new line here should be removed

**Review by limerencee — CHANGES_REQUESTED** [review]
The functionality seem to work with no bugs. I was able to undo and redo many times without any bug occurring. Thanks for the contribution! Should be able to merge after the minor issues are resolved.

File: src/test/java/thrift/logic/commands/AddExpenseCommandTest.java (hunk 2 above)
Comment (lightz96) [reply src/test/java/thrift/logic/commands/AddExpenseCommandTest.java]: Resolved at Commit 6bb04d6

File: src/main/java/thrift/logic/commands/RedoCommand.java (hunk 1 above)
Comment (lightz96) [reply src/main/java/thrift/logic/commands/RedoCommand.java]: Resolved at Commit 6bb04d6

**Review by limerencee — APPROVED** [review]

**Pull request merged**